core/controllers/acl_decorators_test.py

```diff
@@ -1526,6 +1742,13 @@ def setUp(self):
         topic_services.assign_role(
             self.admin, self.manager, topic_domain.ROLE_MANAGER, self.topic_id)
 
+    def test_can_not_edit_topic_with_invalid_topic_id(self):
+        self.login(self.ADMIN_EMAIL)
+        with self.swap(self, 'testapp', self.mock_testapp):
+            self.get_json(
+                '/mock/invalid_topic_id', expected_status_int=404)
```

Comment:
Earlier, I wrote: 

> Also, here and elsewhere, ideally change '/mock/' to be more descriptive as well, e.g. '/mock_publish_exploration/'. Otherwise it's quite hard to follow the tests.

I don't think this has been done. Can you fix it, here and elsewhere?

Reply:
These are the existing test classes that uses 'mock/'. I have made changes to the classes that I created. Should I change these too or can I create an issue to rename the mock URLs?

Reply:
Yes, please change the existing ones too, so that we don't have two different patterns in the codebase.

Reply:
done